### PR TITLE
Blakesager/ch4981/convert leadconduit integrations to es6

### PR DESCRIPTION
**Pull request by chasingSublimity**
Linked to local api and everything seemed ok

Not sure about the lock file here.

ALSO: updated nock, which fixed the tests failing on node 12

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -119,16 +116,20 @@
     "leadconduit-xml": "^1.0.0",
     "lodash": "^4.0.0",
     "markdown-it": "^8.4.1",
+    "nock": "^11.7.2",
```
Comment (cgrayson) [inline package.json]: Nock isn't a `devDependcy`?

Comment (chasingSublimity) [reply package.json]: whoops, definitely is. good catch

**Comment by codingmatty** [comment]
# [Codecov](https://codecov.io/gh/activeprospect/leadconduit-integrations/pull/136?src=pr&el=h1) Report
> :exclamation: No coverage uploaded for pull request base (`master@6ab0d2a`). [Click here to learn what that means](https://docs.codecov.io/docs/error-reference#section-missing-base-commit).
> The diff coverage is `n/a`.

[Impacted file tree graph](https://codecov.io/gh/activeprospect/leadconduit-integrations/pull/136?src=pr&el=tree)

```diff
@@            Coverage Diff            @@
##             master     #136   +/-   ##
=========================================
  Coverage          ?   97.38%           
=========================================
  Files             ?        1           
  Lines             ?      153           
  Branches          ?        0           
=========================================
  Hits              ?      149           
  Misses            ?        4           
  Partials          ?        0           
```



------

[Continue to review full report at Codecov](https://codecov.io/gh/activeprospect/leadconduit-integrations/pull/136?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/activeprospect/leadconduit-integrations/pull/136?src=pr&el=footer). Last update [6ab0d2a...a86b4a8](https://codecov.io/gh/activeprospect/leadconduit-integrations/pull/136?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

**Comment by mikebetts** [comment]
Not sure about the lock file either - will that affect deploying updates to integrations?

And can we get rid of "codingmatty"?

**Comment by chasingSublimity** [comment]
re: codingmatty, think that's a GH bug. The content of that message is a codecov report. Not sure how we'd get rid of that.

**Comment by chasingSublimity** [comment]
Oh, i guess i can just delete it

**Comment by chasingSublimity** [comment]
Regarding lock files, i think this will force a full deploy anytime an integration needs to be updated. That's lame.

However, seems like we're headed that way anyways? https://activeprospect.slack.com/archives/G9M3L7YLD/p1565889005002200

It's simple enough to remove, though. Thoughts, @mikebetts?

**Comment by chasingSublimity** [comment]
@mikebetts I just removed it  👍

**Comment by codingmatty** [comment]
# [Codecov](https://codecov.io/gh/activeprospect/leadconduit-integrations/pull/136?src=pr&el=h1) Report
> :exclamation: No coverage uploaded for pull request base (`master@6ab0d2a`). [Click here to learn what that means](https://docs.codecov.io/docs/error-reference#section-missing-base-commit).
> The diff coverage is `n/a`.

[Impacted file tree graph](https://codecov.io/gh/activeprospect/leadconduit-integrations/pull/136?src=pr&el=tree)

```diff
@@            Coverage Diff            @@
##             master     #136   +/-   ##
=========================================
  Coverage          ?   97.38%           
=========================================
  Files             ?        1           
  Lines             ?      153           
  Branches          ?        0           
=========================================
  Hits              ?      149           
  Misses            ?        4           
  Partials          ?        0           
```



------

[Continue to review full report at Codecov](https://codecov.io/gh/activeprospect/leadconduit-integrations/pull/136?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/activeprospect/leadconduit-integrations/pull/136?src=pr&el=footer). Last update [6ab0d2a...212affa](https://codecov.io/gh/activeprospect/leadconduit-integrations/pull/136?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

**Pull request merged**